entries/b/bendigobank.com.au.json

```
@@ -1,16 +1,18 @@
 {
   "Bendigo Bank": {
     "domain": "bendigobank.com.au",
-    "contact": {
-      "facebook": "BendigoBank",
-      "twitter": "bendigobank",
-      "form": "https://www.bendigoadelaide.com.au/contact-us/customer-feedback/"
-    },
+    "tfa": [
+      "custom-software"
+    ],
+    "custom-software": [
+      "Bendigo Bank app"
+    ],
+    "documentation": "https://www.bendigobank.com.au/security/e-banking-upgrade/#tab-241991",
```

```suggestion
    "documentation": "https://www.bendigobank.com.au/support/e-banking/multi-factor-authentication",
```

I think this is a better page to use for documentation

---

It didn't feel focused enough for me but I'm happy to change it.